### PR TITLE
Monolog Cookbook Typo Fix: "allows to" should be "allows you to"

### PR DESCRIPTION
**Pull request by mattjanssen**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | none |

### CODE REVIEW

File: cookbook/logging/monolog.rst [hunk 1]
```diff
@@ -220,7 +220,7 @@ easily. Your formatter must implement
 Adding some extra Data in the Log Messages
 ------------------------------------------
 
-Monolog allows to process the record before logging it to add some
+Monolog allows you to process the record before logging it to add some
```
Comment (xabbuh) [inline cookbook/logging/monolog.rst]: I'm not a native speaker, but I think this was correct before.

Comment (mattjanssen) [inline cookbook/logging/monolog.rst]: Here's a quick discussion on the phrase "allows to":

http://english.stackexchange.com/questions/85069/is-the-construction-it-allows-to-proper-english

Comment (mattjanssen) [inline cookbook/logging/monolog.rst]: Another option would be "Monolog allows processing of the record before..."

**Comment by WouterJ** [comment]
Let's see what our native speaker @weaverryan thinks about this

**Comment by mattjanssen** [comment]
British or US English?

**Comment by xabbuh** [comment]
@mattjanssen AE

**Comment by weaverryan** [comment]
Both sound ok and so are probably correct. But this new change is correcter. Ahem, I mean more correct. Thanks!

**Pull request closed without merging**